src/pack.rs

```diff
@@ -209,6 +277,11 @@ pub async fn pack(options: PackOptions) -> Result<()> {
             .await
             .map_err(|e| anyhow!("could not copy file to channel directory: {}", e))?;
 
+        if let Some(parent) = path.parent()
+            && parent == output_folder.path()
+        {
+            fs::remove_file(&path).await.ok();
```

Comment:
I don't like this. How about building to a temporary directory instead and letting the `Drop` implementation remove it automatically?